frontend/src/lib/utils/geometry.ts

```diff
@@ -18,145 +18,161 @@ export type Rect = {
 export const ORIGIN = Object.freeze({ x: 0, y: 0 });
 export const MANHATTAN_LENGTH = 13.11;
 
-export const pointerEventToPoint = (event: PointerEvent): Point => {
+export function pointerEventToPoint(event: PointerEvent): Point {
     return { x: event.pageX, y: event.pageY };
-};
+}
 
-export const vectorLength = (vector: Point): number => {
+export function vectorLength(vector: Point): number {
```

Comment:
Should `Vector2D` be a new type? It would of course be a type with `x: number` and `y: number`, but for clarity it is not a `Point` with x- and y-coordinate, but a length/size in x and y.

Reply:
Renamed a couple of functions, introduced `Vector2D`, and appended `2D` to other types for clarity.